<?xml version="1.0"?>
<launch>

    <param name="maxlen"        type="double" value="200.0" />    
    <param name="minlen"        type="double" value="0.0" />    
    <param name="max_ang_FOV"   type="double" value="2.7" />
    <param name="min_ang_FOV"   type="double" value="0.0" />
    <!-- min_ang_FOV : left~right angle of center line -->
    <param name="pcTopic"       type="str"    value="/velodyne_points" />
    <!--<param name="imgTopic"      type="str"    value="/darknet_ros/detection_image" />-->
    <param name="imgTopic"      type="str"    value="/usb_cam/image_raw" />

    <param name="x_resolution"    type="double" value="0.5" />
    <param name="y_interpolation" type="double" value="5.0" />
    <param name="ang_Y_resolution"    type="double" value="1.05" />

    <group ns="matrix_file">
        <rosparam file="$(find lidar_camera_fusion_blue)/cfg/cfg_params.yaml" />
    </group>
    <remap from="/points2" to="/points2_blue"/>
    <remap from="/pcOnImage_image" to="/pcOnImage_image_blue"/>
    <node pkg="lidar_camera_fusion_blue" type="lidar_camera_node_blue" name="lidar_camera_node_blue" output="screen"/>
    <!--<node type="rqt_reconfigure" name="rqt_reconfigure" pkg="rqt_reconfigure"/> -->
<!-- rviz- -->
<node type="rviz" name="rviz_blue" pkg="rviz" args="-d $(find lidar_camera_fusion_blue)/rviz/lidar_camera.rviz" />
  
</launch>
